misk/src/main/kotlin/misk/digester/MergingDigest.kt

```diff
@@ -0,0 +1,351 @@
+package misk.digester
+
+import misk.proto.tdigest.Centroid
+import misk.proto.tdigest.MergingDigestData
+import kotlin.math.PI
+import kotlin.math.asin
+
+class MergingDigest {
+
+    //amount of compression to data
```

Comment:
Style nit: space at the beginning of comments (`// Amount of compression to data`)